DesignSafe-CI/portal
# refactor: des-2737 load app css globally

## Overview: ##

Load app css patterns globally, not per plugin.

## PR Status: ##

* Work in Progress.
* Hold.

## Related Jira tickets: ##

* [DES-2737](https://tacc-main.atlassian.net/browse/DES-2737)

## Summary of Changes: ##

- **removed** per-plugin stylesheets
- **added** app stylesheets back to base styles

## Testing Steps: ##

1. Update prod test snippet.
2. Open reference pages:
    - https://www.designsafe-ci.org/rw/tools-applications/adcirc-backup/?edit
    - https://www.designsafe-ci.org/rw/tools-applications/tools-applications-new-backup/?edit
3. Verify ref pages load all the stylesheets once:
    - `app-grid.css`
    - `app-version-list.css`
    - `app-card.css`
    - `app-page.css`

## UI Photos: ##

| network tab | in markup | css imports |
| - | - | - |
| <img width="500" alt="global css - network tab" src="https://github.com/DesignSafe-CI/portal/assets/62723358/e526b5e2-cffd-4556-adf6-2f1e5a3ebf23"> | <img width="500" alt="global css - markup" src="https://github.com/DesignSafe-CI/portal/assets/62723358/e74e2606-86df-4168-9161-2b2d77c4d02c"> | <img width="500" alt="global css - css import" src="https://github.com/DesignSafe-CI/portal/assets/62723358/930bca23-2da7-4e2c-affc-380ff7cf6701"> |
